DisguiseUnityRenderStream/Editor/DisguiseRenderStreamBuild.cs

```diff
@@ -106,7 +106,7 @@ static void AddSceneToSchema(ManagedSchema schema, int sceneIndex, string name)
             var currentScene = schema.scenes[sceneIndex];
 
             var parameters = currentScene.parameters
-                .Concat(Object.FindObjectsOfType<DisguiseRemoteParameters>()
+                .Concat(Object.FindObjectsByType<DisguiseRemoteParameters>(FindObjectsSortMode.InstanceID)
```

Comment:
`FindObjectsByType` is available since 2020.3 so I didn't bother with conditional compilation for it